src/NuGet.Clients/NuGet.PackageManagement.UI/Options/PackageItem.cs

```diff
@@ -0,0 +1,72 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.ComponentModel.Design;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Collections;
+using NuGet.VisualStudio.Internal.Contracts;
+
+namespace NuGet.Options
+{
+    public class PackageItem
```

Comment:
Can we rename this to be more specific? "Package" and "Item" are very overused in our repo already.

Reply:
Yes I changed it

Reply:
Changing `MappingUIDisplay` -> `SourceMappingViewModel`